core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -189,9 +258,14 @@ public void drawGameOverScreen() {
 
     @Override
     public void dispose() {
-        texture.dispose();
-        pixmap.dispose();
-        gameSpriteBatch.dispose();
+        try {
+            texture.dispose();
+            gameSpriteBatch.dispose();
+            pixmap.dispose();
+        } catch (Exception e){
```

Comment:
this is not necessary. That would happen anyway in the case of an exception

Reply:
Deleted this try/catch block.